Cargo.toml

```diff
@@ -63,7 +63,7 @@ serde_json = { version = "1.0" }
 # "stdlib"
 thiserror = { version = "2.0.11" }
 bytes = { version = "1" }
-chrono = { version = "0.4" }
+chrono = { version = "0.4.39" }
```

Comment:
Fixed patch version for chrono due to https://www.reddit.com/r/rust/comments/1iz0py1/crate_build_is_broken_by_chrono_and_arrowarinth/